include/boost/geometry/algorithms/correct.hpp

```diff
@@ -147,7 +147,13 @@ struct correct_ring
         typedef typename area_result<Ring, Strategy>::type area_result_type;
         Predicate<area_result_type> predicate;
         area_result_type const zero = 0;
-        if (predicate(ring_area_type::apply(r, strategy), zero))
+        if (predicate(ring_area_type::apply(r,
+											// TEMP - in the future (umbrella) strategy will be passed
+			                                geometry::strategies::area::services::strategy_converter
+												<
+													Strategy
+												>::get(strategy)),
```

Comment:
I don't know why but it seems my IDE put tabs instead of spaces. I'll fix it tomorrow. And btw you can see here how to convert simple strategy to umbrella strategy if needed.

Reply:
OK, no problem - and indeed, I notice the converter, really convenient